lib/freeclimb/utils/request_verifier.rb

```diff
@@ -0,0 +1,57 @@
+module Freeclimb
+    class RequestVerifier
+        @@DEFAULT_TOLERANCE = 5 * 60 * 1000
+        attr_accessor :info
+
+        def verify_request_signature(request_body, request_header, signing_secret, tolerance=DEFAULT_TOLERANCE)
+            check_request_body(request_body)
+            check_request_header(request_header)
+            check_signing_secret(signing_secret)
+            check_tolerance(tolerance)
+            @info = Freeclimb::SignatureInformation.new(request_header)
```

Comment:
Does not need to be an instance variable.

Reply:
will remove in latest commit